tests/test_license.py

```diff
@@ -98,6 +98,7 @@ def setUp(self):
             'src/opendr/simulation/human_model_generation/utilities/PIFu',
             'src/opendr/perception/multimodal_human_centric/rgbd_hand_gesture_learner/algorithm/architectures',
             'src/opendr/perception/skeleton_based_action_recognition/algorithm',
+            'projects/data_generation/synthetic_multi_view_facial_image_generation',
```

Comment:
The testing of the license should only be skipped for the algorithm sub-folder, not for the files you created yourself.

Reply:
the folder /algorithm/ is added